scripts/createVPNConfig.sh

```diff
@@ -3,7 +3,13 @@
 # Usage: ./createVPNConfig.sh <environment>
 # Example: ./createVPNConfig.sh staging
 ENVIRONMENT=$1
+if [ "$ENVIRONMENT" == "production" ]; then
+  VAULT=ppnxsriom3alsxj4ogikyjxlzi
+else
+  VAULT=4eyyuwddp6w4vxlabrr2i2duxm
+fi
 git clone https://github.com/cds-snc/notification-terraform.git /var/tmp/notification-terraform
+op read op://$VAULT/"TFVars - $ENVIRONMENT"/notesPlain > /var/tmp/notification-terraform/aws/$ENVIRONMENT.tfvars   
```

Comment:
Technically useful - I didn't include error handling because there's nothing to handle - whether it fails here or on the next step, it will still fail.